### DISCUSSION TITLE
增加mod

### DISCUSSION THREAD

**Discussion by HuangTangyan** [Meloong-Git/PCL#1967] [提问] [answered]
请问能不能把BC管道加上

**Comment by WForst-Breeze** [comment, marked as answer]
只要是上传到 Curseforge 或 Modrinth 的 Mod 都可以在 PCL2 中搜索到，如果没能搜索到，请查看您的搜索关键词或查看该 Mod 是否被上传至上述网站中。

**Reply by Big-Cake-jpg** [reply]
有一个例外：如果 Mod 不符合 Modrinth 的合规性要求，会被从搜索结果中移除，但通过网页链接仍然可以直接访问并下载。
